v3/src/components/map/components/codap-map.tsx

```diff
@@ -17,13 +16,12 @@ import "./map.scss"
 interface IProps {
   mapController: MapController
   mapRef: MutableRefObject<HTMLDivElement | null>
-  dotsRef: IDotsRef
-  dotsElement: SVGSVGElement | null
 }
 
-export const CodapMap = observer(function CodapMap({mapController, mapRef, dotsRef, dotsElement}: IProps) {
+export const CodapMap = observer(function CodapMap({mapController, mapRef}: IProps) {
   const instanceId = useInstanceIdContext(),
-    mapModel = useMapModelContext()
+    mapModel = useMapModelContext(),
+    interiorSvgRef = createRef<SVGSVGElement>()
```

Comment:
```suggestion
    interiorSvgRef = useRef<SVGSVGElement>(null)
```
would be more conventional.